### ISSUE TITLE
Add 'Projects' view alongside Task, Calendar

### ISSUE THREAD

**Issue by jakec-dev** [kdheepak/taskwarrior-tui#266]
**Description**

I often refer to my list of projects using `task projects [filter]`. As far as I'm aware, there is no way to see this report in Taskwarrior-tui so I need to exit out of tui every time I need to run the report.

It would be very helpful and convenient if this functionality were built into tui.

**Solution**

Add a "Projects" tab at the top of the screen (where you can switch between Task and Calendar view).

Selecting this would display your list of projects in the top window (same output as running `task projects`), and the second window (that usually shows task details) could all tasks within the selected project. The bottom "Filter" window could be used to filter the list of projects in the top window.

**Comment by kdheepak** [comment]
Great idea! It should be fairly easy to implement as well.

**Comment by lkadalski** [comment]
Anyone working on this ?

**Comment by kdheepak** [comment]
I started: https://github.com/kdheepak/taskwarrior-tui/tree/projects

But I have been busy on a few other things. I’ll be able to spend more time on this next month.

**Comment by lkadalski** [comment]
Great, may I help you ?

**Comment by lkadalski** [comment]
What was your idea behind calendar view in project pane? 
I'd do this as list similar to tasks pane where you can:

- just select active project (for task view)
- see each project completion

 What do you think?

**Comment by kdheepak** [comment]
I’m happy to accept PRs. Although, I think right now the code for rendering views is not modular at all. I wanted to rewrite that first. That would make it easier to add more “tabs” / for other people to contribute.

**Comment by lkadalski** [comment]
Nevertheless, I think I can make this feature. But I need some guideline from you guys how it should look at the end.
Is that list idea is ok for start ?

**Comment by kdheepak** [comment]
Yeah I think that’sa good start! Thank you to volunteering to contribute!

**Comment by lkadalski** [comment]
Cool, I'm working on this :+1:
